examples/with_pyspark/requirements.txt

```diff
@@ -0,0 +1,3 @@
+-e with_pyspark
+# It's recommended to include environment-specific requirements in this file.
```

Comment:
Comments should be on top or removed I think

Reply:
make sense. this applies to all other examples. i'll clean it up all at once